packages/garbo/src/clan.ts

```diff
@@ -201,6 +205,14 @@ export class StashManager {
             slots.forEach((s) => equip(s, $item.none));
           }
 
+          if (toSlot(item) === $slot`familiar` && !have(item, count)) {
+            for (const familiar of $familiars.all().filter(have)) {
+              if (familiarEquippedEquipment(familiar) === item) {
+                equip(familiar, $item.none);
+              }
+            }
```

Comment:
Now I may be a simple country lawyer but

```ts
$familiars.all().filter(have).filter(f => familiarEquippedEquipment(f) === item).forEach(f => equip(f, $item.none))
```

Reply:
not a big `forEach` guy here